events/code_review_events/workflow.py

```diff
@@ -277,24 +273,26 @@ def __init__(self, cache_root):
             self.webserver.register(self.bus)
 
             # Create pulse listener
-            exchanges = []
+            exchanges = collections.defaultdict(list)
             if taskcluster_config.secrets["autoland_enabled"]:
                 logger.info("Autoland ingestion is enabled")
-                exchanges += [
-                    # autoland ingestion
+                # autoland ingestion
+                exchanges[QUEUE_PULSE_AUTOLAND] = [
                     (PULSE_TASK_GROUP_RESOLVED, ["#.gecko-level-3.#"])
                 ]
             if publish:
                 # unit test failures
-                exchanges += [(PULSE_TASK_COMPLETED, ["*.*.gecko-level-3._"])]
+                exchanges[QUEUE_PULSE_TRY_TASK_END] = [
+                    (PULSE_TASK_COMPLETED, ["*.*.gecko-level-3._"])
+                ]
```

Comment:
I think this was an effect of some bad rebase, we can remove the lines from `if publish:` to here.

Reply:
Done